pkg/machinery/config/types/v1alpha1/generate/init.go

```diff
@@ -77,6 +77,7 @@ func initUd(in *Input) (*v1alpha1.Config, error) {
 		ClusterCA:                     in.Certs.K8s,
 		BootstrapToken:                in.Secrets.BootstrapToken,
 		ClusterAESCBCEncryptionSecret: in.Secrets.AESCBCEncryptionSecret,
+		NoScheduleMasters:             true,
```

Comment:
I think we might need to inverse this boolean option, so that existing Talos installs don't get reverted back to missing `NoSchedule`, in other words `NoSchedule` taint should be the default zero value of the config

Reply:
Good point. Will update